### PR TITLE
Replaced references to the Gitter room with matrix

### PR DESCRIPTION
**Pull request by AnonProductions**
Suucessfully replaced the hyperlinks on the files mentioned

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,4 +51,4 @@ Now you can just head over to `http://localhost:4000` and see your live version
 
 Contributing to the opsdroid ecosystem is strongly encouraged and every little bit counts! We even send [sticker packs](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59) to our contributors to say thank you! 
 
-Do you need help? Do you want to chat? [Join our Gitter channel](https://gitter.im/opsdroid/)
\ No newline at end of file
+Do you need help? Do you want to chat? [Join our Gitter channel](https://riot.im/app/#/room/#opsdroid-general:matrix.org)
```
Comment (jacobtomlinson) [inline README.md]: Can you also change the text.
```suggestion
Do you need help? Do you want to chat? [Join our Matrix channel](https://riot.im/app/#/room/#opsdroid-general:matrix.org).
```

File: _includes/footer.html [hunk 2]
```diff
@@ -8,37 +8,43 @@ <h1 class="add-shadow">Latest Posts</h1>
...
       <p class="white-text align-left">Don't miss a thing</p>
       <div class="align-left social-icons">
-          <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i class="fab fa-github fa-lg"></i></a>
-          <a href="https://gitter.im/opsdroid" title="Opsdroid Gitter Channel"><i class="fab fa-gitter fa-lg"></i></a>
-          <a href="https://twitter.com/{{ site.twitter_username }}" title="Opsdroid Twitter Account"><i class="fab fa-twitter fa-lg"></i></a>
-          <a href="https://medium.com/opsdroid" title="Opsdroid Medium Blog"><i class="fab fa-medium-m fa-lg"></i></a>
-          <a href="https://www.youtube.com/channel/UC0FeGkDF7FrshmzAsb9lDvw" title="Opsdroid Youtube Account"><i class="fas fa-caret-right fa-lg"></i></a>
+        <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i
+            class="fab fa-github fa-lg"></i></a>
+        <a href="https://riot.im/app/#/room/#opsdroid-general:matrix.org" title="Opsdroid Gitter Channel"><i
```
Comment (jacobtomlinson) [inline _includes/footer.html]: Text also needs changing here

```suggestion
        <a href="https://riot.im/app/#/room/#opsdroid-general:matrix.org" title="Opsdroid Matrix Channel"><i
```

File: _includes/footer.html [hunk 3]
```diff
@@ -8,37 +8,43 @@ <h1 class="add-shadow">Latest Posts</h1>
...
+            class="fab fa-gitter fa-lg"></i></a>
```
Comment (jacobtomlinson) [inline _includes/footer.html]: This is still using the gitter icon. Ther currently isn't a Matrix or Riot icon in font awesome so maybe a chat style icon like `comment-alt` would be suitable.

```suggestion
            class="fab fa-comment-alt fa-lg"></i></a>
```

**Review by jacobtomlinson — CHANGES_REQUESTED** [review]
Thanks so much for raising this! 

It looks like you've updated the links but not the text or icons which still show as Gitter. Could you please update these too as per my comments.

**Comment by AnonProductions** [comment]
Could you please check it out now? I've changed the necessary text and icons

File: _includes/footer.html [hunk 4]
```diff
@@ -8,37 +8,43 @@ <h1 class="add-shadow">Latest Posts</h1>
...
       <div class="align-left social-icons">
-          <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i class="fab fa-github fa-lg"></i></a>
-          <a href="https://gitter.im/opsdroid" title="Opsdroid Gitter Channel"><i class="fab fa-gitter fa-lg"></i></a>
-          <a href="https://twitter.com/{{ site.twitter_username }}" title="Opsdroid Twitter Account"><i class="fab fa-twitter fa-lg"></i></a>
-          <a href="https://medium.com/opsdroid" title="Opsdroid Medium Blog"><i class="fab fa-medium-m fa-lg"></i></a>
-          <a href="https://www.youtube.com/channel/UC0FeGkDF7FrshmzAsb9lDvw" title="Opsdroid Youtube Account"><i class="fas fa-caret-right fa-lg"></i></a>
+        <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i
+            class="fab fa-github fa-lg"></i></a>
+        <a href="https://riot.im/app/#/room/#opsdroid-general:matrix.org" title="Opsdroid Matrix Channel"><i
+            class="fab fa-matrix fa-lg"></i></a>
```
Comment (jacobtomlinson) [inline _includes/footer.html]: There is no such icon `fa-matrix`. Could you please change this to `fa-comment-alt` as I suggested?

**Review by jacobtomlinson — CHANGES_REQUESTED** [review]
Thanks for making these changes. There may have been a misunderstanding with this icon, there is no matrix icon in font awesome.

**Comment by AnonProductions** [comment]
Could you please check it and let me know if there are any further issues?

**Comment by jacobtomlinson** [comment]
This is great many thanks for iterating on this with me!

We love to thank contributors by sending them [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)! Fill in the [claim form](https://forms.gle/iQZV9DjkaUiSaDXK6) to get yours!

**Comment by AnonProductions** [comment]
Thanks a lot for the appreciation. Would love to get some stickers :)

**Pull request merged**